apps/app-article/src/components/card.js

```diff
@@ -31,7 +32,13 @@ const Card = (props) => {
                         <div className={"article-premium-badge"}><PremiumBadge/></div>
                         : ''
                 }
-                <img className={"card-img-top"} src={props.article.listImage.url + "&function=hardcrop&width=798&height=649&q=95"}/>
+                {
+                    props.article.listImage.url != null
```

Comment:
Let's use `!!props.article.listImage.url ? ... : ...` here, as this will prevent `null`, `undefined` and empty strings to be evaluated true.

Reply:
thank you, this is way better!